_addons_transformations/scale/readme.md

```diff
@@ -14,17 +14,47 @@ install: auto
 
 # Scale Transformation Service
 
-Transform the input by matching it between limits of ranges in a scale file.  The input string must be in numerical format.
+Transform a given input by matching it between limits of ranges.
+The input string must be in numerical format.
 
-The file is expected to exist in the `transform` directory, and should follow the format given in the example below.
+The file is expected to exist in the `transform` directory, and should follow the format given in the table below.
```

Comment:
Please mention, that the file ending has to be scale and that the directory can be found under the configuration path

Reply:
Also please add one sentence describing the left and right part of the expression